scripts/securemsg.pl

```diff
@@ -516,6 +517,38 @@ sub cmd_show {
     }
 }
 
+sub cmd_clear {
+    my ($args, $server, $witem) = @_;
+    my ($nick, $net) = _get_nn($args);
+    my $server;
+
+    if((!defined $nick) || (!exists $messages{$nick})) {
+        usage;
```

Comment:
175 `print("Usage: sm add|del <nick>|<host> list of nicks/hosts | sm nicks|hosts | sm accept|reject <nick> [net <chatnet>] [message] | sm show <nick> [net <chatnet>] | sm help");`

Reply:
Are you saying usage is printed if you try to use /sm clear <nick> or that usage is missing info on clear?

Reply:
`sub usage` miss the info about clear

Reply:
Thanks! I updated the help but missed that bit :)